programs/survival/bucketstack.sc

```diff
@@ -0,0 +1,133 @@
+__config() ->
+(
+	//change these number to whatever you want the maximum stack size to be
+	//note that the max scarpet can inventory_set() is 2^31 - 1
+	global_buckets = m(
+		l('water',1),
+		l('lava',1),
+		l('milk',1),
+		l('pufferfish',1),
+		l('salmon',1),
+		l('cod',1),
+		l('tropical_fish',1),
+		l('axolotl',1)
+	);
+	return(

```

Comment:
No need for a `return()` call here, pretty sure. And you can use the more readable `[·]` and `{·}` calls intead of `l(·)` and `m(·)`.

Reply:
Return call has been removed

Reply:
While I don't mind changing these, keep in mind that readability of  `[]` and `{}` over `l()` and `m()` is not purely objective. If we have a standard for this repo, I'm happy to follow it.